crates/biome_cli/src/execute/process_file.rs

```diff
@@ -128,6 +128,16 @@ pub(crate) fn process_file(ctx: &TraversalOptions, path: &Path) -> FileResult {
                 category!("files/missingHandler"),
             )?;
 
+        // first we stop if there are some files that don't have ALL features enabled, e.g. images, fonts, etc.
+        if file_features.is_ignored() || file_features.is_not_enabled() {
```

Comment:
Do we use experimental Rust features in Biome? If yes, can we surround this with `std::intrinsics::unlikely`?

Reply:
No, we don't use experimental features, and I don't think we plan for now (at least for stable releases)